### PR TITLE
Add cluster-autoscaler

### PR DESCRIPTION
**Pull request by paurosello**
### Checklist

- [x] Update changelog in CHANGELOG.md.

### Trigger e2e tests

/run cluster-test-suites

### CODE REVIEW

**Review by AndiDog — APPROVED** [review]

**Comment by calvix** [comment]
@paurosello why is aws pod identity webhook needed? AFAIK EKS provide this feature out of the box

File: helm/default-apps-eks/values.yaml [hunk 1]
```diff
@@ -55,6 +65,19 @@ userConfig:
...
+    catalog: default
+    clusterValues:
+      configMap: true
+      secret: false
+    dependsOn: cert-manager  # since aws-pod-identity-webhook installs a `Certificate` object
+    enabled: true
+    namespace: kube-system
+    # used by renovate
+    # repo: giantswarm/aws-pod-identity-webhook
+    version: 1.13.2
```
Comment (calvix) [inline helm/default-apps-eks/values.yaml:80]: Why do we need aws-pod-identity-webhook

**Review by calvix — CHANGES_REQUESTED** [review]
not sure about the aws-pod-identity-webhook

**Comment by weseven** [comment]
@calvix @paurosello I've tested the autoscaler without aws-pod-identity-webhook, it works correctly (both scale up and scale down).

Pending your reviews, this is ready to be merged.

**Review by calvix — APPROVED** [review]

**Pull request merged**